### PR TITLE
docs: concepts -- add information about tool calling models, update tools section

### PR DESCRIPTION
**Pull request by eyurtsev**
- Add information about naitve tool calling capabilities
- Add information about standard langchain interface for tool calling
- Update description for tools

### CODE REVIEW

File: docs/docs/concepts.mdx [hunk 1]
```diff
@@ -128,13 +128,14 @@ LangChain provides standard, extendable interfaces and external integrations for
...
 ### Chat models
+
 Language models that use a sequence of messages as inputs and return chat messages as outputs (as opposed to using plain text).
 These are traditionally newer models (older models are generally `LLMs`, see above).
 Chat models support the assignment of distinct roles to conversation messages, helping to distinguish messages from the AI, users, and instructions such as system messages.
 
-Although the underlying models are messages in, message out, the LangChain wrappers also allow these models to take a string as input.
-This makes them interchangeable with LLMs (and simpler to use).
-When a string is passed in as input, it will be converted to a HumanMessage under the hood before being passed to the underlying model.
+Although the underlying models are messages in, message out, the LangChain wrappers also allow these models to take a string as input. This makes them interchangeable with LLMs.
```
Comment (ccurme) [inline docs/docs/concepts.mdx:136]: nit: not sure if interchangeable implies we can change ChatModels -> LLMs
```suggestion
Although the underlying models are messages in, message out, the LangChain wrappers also allow these models to take a string as input. This means you can easily use chat models in place of LLMs.
```

File: docs/docs/concepts.mdx [hunk 2]
```diff
@@ -143,7 +144,22 @@ We have some standardized parameters when constructing ChatModels:
...
 ChatModels also accept other parameters that are specific to that integration.
 
+#### Tool Calling
+
+Some chat models have been fine-tuned for tool calling and provide a dedicated API for tool calling. 
+Given a list of tools and a prompt, these chat models can request that one or more tools be invoked with given arguments.
+
+Generally, such models are better at tool calling than non-fine-tuned models, and are recommended for use cases that require tool calling.
+
+LangChain provides a standardized interface for tool calling that makes it work with different models.
```
Comment (ccurme) [inline docs/docs/concepts.mdx:154]:
```suggestion
LangChain provides a standardized interface for tool calling that is consistent across different models.
```

File: docs/docs/concepts.mdx [hunk 3]
```diff
@@ -143,7 +144,22 @@ We have some standardized parameters when constructing ChatModels:
 
 ChatModels also accept other parameters that are specific to that integration.
 
+#### Tool Calling
```
Comment (ccurme) [inline docs/docs/concepts.mdx:147]: should we consolidate this with the `#functiontool-calling` section further down?

**Review by ccurme — APPROVED** [review]

File: docs/docs/concepts.mdx (hunk 3 above)
Comment (eyurtsev) [reply docs/docs/concepts.mdx:147]: yes good catch, missed that section!

**Pull request merged**